eng/build.yml

```diff
@@ -117,32 +118,20 @@ phases:
           BlobFeedUrl: $(PB_PublishBlobFeedUrl)
           PublishType: $(_PublishType)
 
-    - task: PublishTestResults@1
-      displayName: Publish Test Results
-      inputs:
-        testRunner: XUnit
-        testResultsFiles: 'artifacts/TestResults/$(_BuildConfig)/*.xml'
-        testRunTitle: '$(_AgentOSName)_$(Agent.JobName)'
-        platform: '$(BuildPlatform)'
-        configuration: '$(_BuildConfig)'
-      condition: not(succeeded())
-
-    - task: CopyFiles@2
-      displayName: Gather Logs
-      inputs:
-        SourceFolder: '$(Build.SourcesDirectory)'
-        Contents: |
-         artifacts/log/$(_BuildConfig)/**/*
-         artifacts/TestResults/$(_BuildConfig)/**/*
-        TargetFolder: '$(Build.ArtifactStagingDirectory)'
-      continueOnError: true
-      condition: not(succeeded())
+    - ${{ if and(eq(parameters.enablePublishBuildAssets, true), eq(parameters.runAsPublic, 'false'), ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
You should be able to delete this section of code as it is handled automatically by `job.yml` since you specified `enablePublishBuildAssets=true` above.

Reply:
Done.